splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -162,6 +162,8 @@ private void configureHandler() throws Exception {
                     .toD(https("$simple{headers.metadata[url].replaceFirst('^https://', '')}/services/collector/event")
                         .httpMethod("POST"))
                     .endChoice()
+                .otherwise()
+                    .to(direct("error"))
```

Comment:
Unfortunately this produces two messages, one with failure (body is the message) and another one as success.
Easiest would be to claim the last choice as "otherwise" branch and leverage exceptions (out of the http(s) component).

Reply:
ok, so just put it back the way we had it before, essentially?